cfme/configure/settings.py

```diff
@@ -20,6 +20,7 @@ class TimeProfileAddForm(View):
     days = BootstrapSwitch(name='all_days')
     hours = BootstrapSwitch(name='all_hours')
     save_button = Button('Save')
+    add = Button('Add')
     configuration = Dropdown('Configuration')
     table = Table("//div[@id='main_div']//table")
     save_edit_button = Button('Save')
```

Comment:
REQUIRED: please check why does this view have two Save buttons

Reply:
The second save button is always save on edit page.

Reply:
as far as I see both Button widgets have Save title. So, I don't see any difference between them.